src/pip/_internal/cli/cmdoptions.py

```diff
@@ -983,6 +983,7 @@ def check_list_path_option(options: Values) -> None:
     choices=[
         "fast-deps",
         "truststore",
+        "no-truststore",
```

Comment:
The `no-truststore` one should be moved to `use-deprecated` (and probably renamed to something like `legacy-ssl`?